.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Feature request
```

Comment:
Is it possible to have a feature request button that just goes to the discussion boards?

Reply:
Doesn't look like Github allows this, can you show me examples of issues created for feature requests? I wonder where they are coming from, regardless we need to make sure we make our channels official

Reply:
Perhaps we should remove the "Feature request" template, it's a misleading template

Reply:
I suspect we get more support requests via this than actual useful suggestions, but I don't monitor the channel. Ask Don :-)

Reply:
> I suspect we get more support requests via this than actual useful suggestions

Correct. 50% is I'm trying to build QGC on some random platforms and it's not working. The rest is bug reports. Almost all of them with no enough information to do anyting about initially.

Feature Requests end up in Issues mostly from questions started on Forums where I asked the person to add a feature request in. Rarely do they start in Issues.

Reply:
I think removing the Feature Request button completely makes a lot of sense.